README.md

```diff
@@ -445,6 +446,12 @@ is a local cache of counters keyed by the encoded metric names. This avoids the
 of encoding the metric name and tags every time, if we already have created a counter for that city.
 It also ensures that we will report separate time-series for order-counts of different city codes.
 
+#### Restrictions on Tags
+Following restrictions should be observed with respect to tag keys and values:
+- **Allowed characters in tag keys** - Tag keys are allowed to have all unicode characters and `.`, `-`, `_`, `/` are allowed.
```

Comment:
should be "Unicode letters (as defined by the Unicode specification)" not "unicode characters"